docs/resources/repository_credential.md

```diff
@@ -0,0 +1,251 @@
+  name = "tf-secret" # Required
+
+  scope {
+    cluster {
+      cluster_name            = "testcluster" # Required
+      provisioner_name        = "attached"    # Default: attached
+      management_cluster_name = "attached"    # Default: attached
+    }
+  }
+
+  meta {
+    description = "Create namespace through terraform"
+    labels      = { "key" : "value" }
+  }
+
+  spec {
+    username_password {
+      username = "testusername" # Required
+      password = "testpassword" # Required
+    }
+  }
+}
+```
+
+## Cluster group scoped Repository Credential with SSH Key type credential
+
+### Example Usage
+
+```terraform
+# Create Tanzu Mission Control source secret with attached set as default value.
+resource "tanzu-mission-control_repository_credential" "create_source_secret" {
+  name = "tf-secret" # Required
+
+  scope {
+    cluster_group {
+      name = "default" # Required
+    }
+  }
+
+  meta {
+    description = "Create namespace through terraform"
+    labels      = { "key" : "value" }
+  }
+
+  spec {
+    ssh_key {
+      identity    = "testidentity"    # Required
+      known_hosts = "testknown_hosts" # Required
+    }
+  }
+}
+```
+
+## Cluster scoped Repository Credential with SSH Key type credential
+
+### Example Usage
+
+```terraform
+# Create Tanzu Mission Control source secret with attached set as default value.
+resource "tanzu-mission-control_repository_credential" "create_source_secret" {
+  name = "tf-secret" # Required
+
+  scope {
+    cluster {
+      cluster_name            = "testcluster" # Required
+      provisioner_name        = "attached"    # Default: attached
+      management_cluster_name = "attached"    # Default: attached
+    }
+  }
+
+  meta {
+    description = "Create namespace through terraform"
+    labels      = { "key" : "value" }
+  }
+
+  spec {
+    ssh_key {
+      identity    = "testidentity"    # Required
+      known_hosts = "testknown_hosts" # Required
+    }
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) Name of the source secret.
+- `scope` (Block List, Min: 1, Max: 1) Scope for the kustomization, having one of the valid scopes: cluster, cluster_group. (see [below for nested schema](#nestedblock--scope))
+- `spec` (Block List, Min: 1, Max: 1) (see [below for nested schema](#nestedblock--spec))
+
+### Optional
+
+- `meta` (Block List, Max: 1) Metadata for the resource (see [below for nested schema](#nestedblock--meta))
+- `org_id` (String) ID of Organization.
+
+### Read-Only
+
+- `id` (String) The ID of this resource.
+
+<a id="nestedblock--scope"></a>
+### Nested Schema for `scope`
+
+Optional:
+
+- `cluster` (Block List, Max: 1) The schema for cluster iam policy full name (see [below for nested schema](#nestedblock--scope--cluster))
+- `cluster_group` (Block List, Max: 1) The schema for cluster group iam policy full name (see [below for nested schema](#nestedblock--scope--cluster_group))
```

Comment:
Please remove all references to policy from this MR. @Shubbhang351

Reply:
Update PTAL